images/nfs/Dockerfile

```diff
@@ -14,37 +14,27 @@
 
 #Portions of this file came from https://github.com/kubernetes-incubator/external-storage/blob/master/nfs/deploy/docker/Dockerfile, which uses the same license.
 
-FROM NFS_BASEIMAGE
-
+FROM ubuntu:xenial
```

Comment:
put `ubuntu:xenial` in `NFS_BASEIMAGE` variable in the [Makefile](https://github.com/rook/rook/blob/master/images/nfs/Makefile#L25) and reference it here